python/tk_desktop2/websockets/util.py

```diff
@@ -7,9 +7,13 @@
 
 import sgtk
 import json
-import collections
 
-from tank_vendor import six
+from collections.abc import Mapping, Iterable
```

Comment:
what about this change? I don't see any other changes where this would be used. That does not sound right.

Reply:
you are right, this change is not part of the first phase scope of the plan, I will revert this

Reply:
Do you know if we provide these libraries under our new `sgutils`?

Reply:
We don't provide those libraries in sgutils.py, the idea is to use python native Collections in phase 3, following the [plan](https://wiki.autodesk.com/pages/viewpage.action?spaceKey=SHOT&title=Plan+execution+Py2)